skin/base-quarantine.css

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
I'll reintegrate it in base.css, as a section at the end if things don't seem to fit in the existing sections.